### ISSUE TITLE
Wrong bundled files on 0.1.3

### ISSUE THREAD

**Issue by mateuswetah** [damienroche/vue-blurhash#9]
Hello again!

Unfortunately, it looks like something is different in your bundled version from `0.1.2` to `0.1.3` on npm. In the previous version, the folder structure was like:
```
vue-blurhash/
|----dist/
|--------vue-blurhash.css
|--------vue-blurhash.common.js
|--------...
```
And now is more like this:
```
vue-blurhash/
|----dist/
|--------css/
|--------js/
|------------app.45a8bcab.js
|------------vendor.45a8bcab.js
|--------...
```

I wasn't able to import it properly using the README instructions with this structure :(

**Comment by damienroche** [comment]
@mateuswetah my bad, I made a big mistake when building bundle. Fixed in 0.1.4

**Comment by mateuswetah** [comment]
No problem, thanks again!
